.github/workflows/python_actions.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       with:
         repository: SpiNNakerManchester/SupportScripts
         path: support
+        ref: spell-chucking
```

Comment:
This line ***MUST*** be deleted before this PR is merged. That can only be done after SpiNNakerManchester/SupportScripts#54 is merged.